pkg/object/risingwave_manager.go

```diff
@@ -126,13 +133,15 @@ func (mgr *RisingWaveManager) UpdateCondition(condition risingwavev1alpha1.Risin
 	}
 }
 
+// UpdateStatus receives a function to mutate the RisingWaveStatus and runs it within a lock.
```

Comment:
For the functions that use `mgr.mu.Lock()`. Should we mention that this is atomic? E.g. 
```suggestion
// UpdateStatus atomically updates the RisingWaveStatus using by applying f
```

Reply:
I did mentioned that it will be run within a lock 😄